google/cloud/storage/bucket_metadata.cc

```diff
@@ -463,6 +466,18 @@ BucketMetadataPatchBuilder& BucketMetadataPatchBuilder::ResetRetentionPolicy() {
   return *this;
 }
 
+BucketMetadataPatchBuilder& BucketMetadataPatchBuilder::SetRpo(
+    std::string const& v) {
+  if (v.empty()) return ResetRpo();
```

Comment:
Returning a reference to a temporary?

Reply:
`ResetRpo()` returns a reference to `*this`, just like all the functions in this class.  I guess `*this` could be a temporary, but that is not too terrible I hope?  That is how all this "fluent" interfaces work?

Reply:
Sorry ... I couldn't even see 5 lines down the page.  :-(